src/lib/requestManager.ts

```diff
@@ -1,9 +1,26 @@
-import axios, { Method as HTTPMethod, ResponseType, AxiosRequestConfig, AxiosResponse } from 'axios';
+import axios, { Method as HTTPMethod, ResponseType, AxiosRequestConfig, AxiosResponse, AxiosError } from 'axios';
 
-export const defaultOptions: { responseType: ResponseType } = {
-  responseType: 'json',
+import { Config } from 'config';
+import { JSONValue, keysToCamelCase } from 'helpers/json';
+
+export const successResponseInterceptor = (
+  response: AxiosResponse<unknown>
+): AxiosResponse<unknown> | Promise<AxiosResponse<unknown>> => {
```

Comment:
Why a sync function has `Promise<AxiosResponse<unknown>>` as returning value?

Reply:
@nvminhtue my bad, will remove it